Don't try and install as root.
Always set LIBDIR to the include the DESTDIR, otherwise it uses the
incorrect value set in the toplevel makefile.

Index: pptpd-1.3.4/plugins/Makefile
===================================================================
--- pptpd-1.3.4.orig/plugins/Makefile	2006-08-03 12:02:01.000000000 +1000
+++ pptpd-1.3.4/plugins/Makefile	2007-06-09 11:42:57.000000000 +1000
@@ -3,7 +3,7 @@
 CFLAGS	= $(COPTS) -I.. -I../../include -fPIC
 LDFLAGS	= -shared
 LDADD   = -lutil
-INSTALL	= install -o root
+INSTALL	= install
 prefix  = /usr/local
 
 PLUGINS = pptpd-logwtmp.so
@@ -18,7 +18,7 @@
 %.so: %.c
 	$(CC) -o $@ $(LDFLAGS) $(CFLAGS) $^ $(LDADD)
 
-LIBDIR	?= $(DESTDIR)$(prefix)/lib/pptpd
+LIBDIR	= $(DESTDIR)$(prefix)/lib/pptpd
 
 install: $(PLUGINS)
 	$(INSTALL) -d $(LIBDIR)
